### ISSUE TITLE
New Amon Variable: Freshwater Flux

### ISSUE THREAD

**Issue by Funkensieper** [PCMDI/obs4MIPs-cmor-tables#72]
After the remaining DWD variables have been signed off, we would like to propose a new Amon variable for Freshwater Flux:

        "fr":{
            "cell_measures":"area: areacella",
            "cell_methods":"area: time: mean",
            "comment":"Difference of Precipitation and Evaporation",
            "dimensions":"longitude latitude time",
            "frequency":"mon",
            "long_name":"Freshwater Flux",
            "ok_max_mean_abs":"",
            "ok_min_mean_abs":"",
            "out_name":"fr",
            "positive":"",
            "standard_name":"",
            "type":"real",
            "units":"kg m-2 s-1",
            "valid_max":"",
            "valid_min":""
        }

**Comment by taylor13** [comment]
Check this, but I think the standard name for this is: surface_downward_water_flux
which is described as 
```
The surface called "surface" means the lower boundary of the 
atmosphere. "Water" means water in all phases, including frozen 
i.e. ice and snow. "Downward" indicates a vector component 
which is positive when directed downward (negative upward). 
The surface water flux is the result of precipitation and 
evaporation. In accordance with common usage in geophysical 
disciplines, "flux" implies per unit area, called "flux density" in 
physics."
```
I might also define the "longname" as "Surface Downward Freshwater Flux" to be more specific (or leave out "freshwater").

As for the variable name and outname, consider using "pme" (precip. - evap.), which has been used in some past mips, or perhaps to make it explicit that you are considering the interface between the surface and atmosphere "pmes" (precip. - evap. at surface).

@gleckler1 might have some opinion about this.

**Comment by durack1** [comment]
@Funkensieper it may also be a good idea to consider the existing `wfo` variable which is defined in the `Omon` table, see [here](https://github.com/PCMDI/obs4MIPs-cmor-tables/blob/master/Tables/obs4MIPs_Omon.json#L4675-L4691)

**Comment by Funkensieper** [comment]
Thanks for the suggestions! I'll discuss them with the dataset experts and come back to you next week.

**Comment by Funkensieper** [comment]
Unfortunately, ``wfo`` is not suitable because it also includes river inflow and sea ice effects (see the ``water_flux_into_sea_water`` standard name) which is not the case for our dataset. So we would go for ``pmes``:

    "pmes":{
        "cell_measures":"area: areacella",
        "cell_methods":"area: time: mean",
        "comment":"Difference of Precipitation and Evaporation",
        "dimensions":"longitude latitude time",
        "frequency":"mon",
        "long_name":"Surface Downward Freshwater Flux",
        "ok_max_mean_abs":"",
        "ok_min_mean_abs":"",
        "out_name":"pmes",
        "positive":"",
        "standard_name":"surface_downward_water_flux",
        "type":"real",
        "units":"kg m-2 s-1",
        "valid_max":"",
        "valid_min":""
    }

**Comment by durack1** [comment]
@Funkensieper apologies for the delay in replying.. Can you add in some further description?

As an example below (note this version is from the upstream cmip6-cmor-tables source), here is the `Amon` entry for surface evaporation - see [here](https://github.com/PCMDI/obs4MIPs-cmor-tables/blob/master/Tables/obs4MIPs_Amon.json#L494-L510). We'll also likely add back in the realm info (like the upstream version, see [here](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_Amon.json#L376-L393) as I can imagine without the realm it will be hard to ascertain what the quantity actually is:
```json
        "evspsbl": {
            "frequency": "mon", 
            "modeling_realm": "atmos", 
            "standard_name": "water_evaporation_flux", 
            "units": "kg m-2 s-1", 
            "cell_methods": "area: time: mean", 
            "cell_measures": "area: areacella", 
            "long_name": "Evaporation", 
            "comment": "Evaporation at surface: flux of water into the atmosphere due to conversion of both liquid and solid phases to vapor (from underlying surface and vegetation)", 
            "dimensions": "longitude latitude time", 
            "out_name": "evspsbl", 
            "type": "real", 
            "positive": "up", 
            "valid_min": "", 
            "valid_max": "", 
            "ok_min_mean_abs": "", 
            "ok_max_mean_abs": ""
},
```

**Comment by Funkensieper** [comment]
What about

    "comment": "Freshwater flux at surface: net flux of fresh water resulting from the difference of precipitation and evaporation."

?

**Comment by taylor13** [comment]
in some contexts I think evaporation excludes sublimation (and maybe even evapo-transpiration).  Both of those should be included.  Also when water vapor condenses onto a cold surface ("dew formation") that wouldn't normally be considered "precipitation" because it is neither liquid or solid water falling onto the surface.  So although your comment would probably be well understood by most, it may not be as explicit as we would like.

**Comment by Funkensieper** [comment]
Ok, next try :)

"Freshwater flux at surface: net flux of fresh water between the atmosphere and the underlying surface (and vegetation), mainly resulting from the difference of precipitation and evaporation. In general, all physical processes, which transport water between the atmosphere and the underlying surface and vegetation, are included."

**Comment by taylor13** [comment]
Although a bit wordy, this I think is fine.  A bit shorter, but perhaps not any easier to understand: 

comment = "net flux of water (in all phases) between the atmosphere and underlying surface (including vegetation), mainly resulting from the difference of precipitation and evaporation."

But your suggestion is now explicit, so its up to you. 

By the way, why is "freshwater" used in your long name and comment?  Wouldn't "water" be more accurate?   I would think this flux should include "sea spray" which subsequently evaporates.  Isn't sea spray salt water?  I would recommend replacing "freshwater" with "water".

**Comment by Funkensieper** [comment]
Agree to your formulation, thanks! 

Regarding water/freshwater: Even if water evaporates from sea spray it's still freshwater. Anyway, our dataset only covers freshwater, so we would like to stick to that.

**Comment by taylor13** [comment]
Great.  Thanks.   I think this issue can be closed.

**Comment by durack1** [comment]
@taylor13 we'll actually need to implement the changes described above before the issue can be closed

**Comment by Funkensieper** [comment]
And, for completeness, the final variable definition:

    "pmes":{
        "cell_measures":"area: areacella",
        "cell_methods":"area: time: mean",
        "comment":"Net flux of water (in all phases) between the atmosphere and underlying surface (including vegetation), mainly resulting from the difference of precipitation and evaporation.",
        "dimensions":"longitude latitude time",
        "frequency":"mon",
        "long_name":"Surface Downward Freshwater Flux",
        "ok_max_mean_abs":"",
        "ok_min_mean_abs":"",
        "out_name":"pmes",
        "positive":"",
        "standard_name":"surface_downward_water_flux",
        "type":"real",
        "units":"kg m-2 s-1",
        "valid_max":"",
        "valid_min":""
    }

**Comment by taylor13** [comment]
What does it take to implement?

**Comment by durack1** [comment]
@taylor13 I need to incorporate the changes documented above https://github.com/PCMDI/obs4MIPs-cmor-tables/issues/72#issuecomment-330824158, generate new tables, and close this issue

**Comment by durack1** [comment]
@gleckler1 would you like to intervene with the `pmes` name description?

**Comment by gleckler1** [comment]
I presume the "s" in "pmes" is for surface.   While that is ok, "pme" is a well-know acronym, almost as much as "TOA" ... either is okbyme

**Comment by taylor13** [comment]
Yes, I think "pme" is the more common usage.

**Comment by durack1** [comment]
Reopened as `cell_methods` requires tweaking [PCMDI/obs4MIPs-cmor-tables#95](https://github.com/PCMDI/obs4MIPs-cmor-tables/pull/95#issuecomment-333463153):

@Funkensieper "pme's cell methods should be changed to area:time:mean. Sorry, I mixed that up in my previous comment. Besides, the cell methods of clCCI, cltCCI, clwCCI, clwtCCI, pctCCI were accidentally changed by commit 9bbb576 . The correct cell_methods here are area:time:mean."
